armi/physics/neutronics/tests/test_crossSectionManager.py

```diff
@@ -209,6 +209,117 @@ def test_ComponentAverageRepBlock(self):
         )
 
 
+class TestBlockCollectionComponentAverage1DCylinder(unittest.TestCase):
+    r"""
+    tests for 1D cylinder XS gen cases
+    """
+
+    def setUp(self):
+        r"""
+        First part of setup same as test_Cartesian.
+        Second part of setup builds lists/dictionaries of expected values to compare to.
+        has expected values for component isotopic atom density and component area
+        """
+        self.o, self.r = test_reactors.loadTestReactor(TEST_ROOT)
+
+        sodiumDensity = {"NA23": 0.022166571826233578}
+        steelDensity = {
+            "C": 0.0007685664978992269,
+            "V": 0.0002718224847461385,
+            "SI28": 0.0003789374369638149,
+            "SI29": 1.924063709833714e-05,
+            "SI30": 1.268328992580968e-05,
+            "CR50": 0.0004532023742335746,
+            "CR52": 0.008739556775111474,
+            "CR53": 0.0009909955713678232,
+            "CR54": 0.000246679773317009,
+            "MN55": 0.0004200803669857142,
+            "FE54": 0.004101496663229472,
+            "FE56": 0.06438472483061823,
+            "FE57": 0.0014869241111006412,
+            "FE58": 0.00019788230265709334,
+            "NI58": 0.0002944487657779742,
+            "NI60": 0.00011342053328927859,
+            "NI61": 4.930763373747379e-06,
+            "NI62": 1.571788956157717e-05,
+            "NI64": 4.005163933412346e-06,
+            "MO92": 7.140180476114493e-05,
+            "MO94": 4.4505841916481845e-05,
+            "MO95": 7.659816252004227e-05,
+            "MO96": 8.02548587207478e-05,
+            "MO97": 4.594927462728666e-05,
+            "MO98": 0.00011610009956095838,
+            "MO100": 4.6334190016834624e-05,
+            "W182": 3.663619370317025e-05,
+            "W183": 1.9783544599711936e-05,
+            "W184": 4.235973352562047e-05,
+            "W186": 3.9304414603061506e-05,
+        }
+        linerAdjustment = 1.014188527784268
+        cladDensity = {
+            nuc: dens * linerAdjustment for nuc, dens in steelDensity.items()
+        }
+        fuelDensity = {
+            "AM241": 2.3605999999999997e-05,
+            "PU238": 3.7387e-06,
+            "PU239": 0.0028603799999999996,
+            "PU240": 0.000712945,
+            "PU241": 9.823120000000004e-05,
+            "PU242": 2.02221e-05,
+            "U235": 0.00405533,
+            "U238": 0.0134125,
+        }
+        self.expectedComponentDensities = [
+            fuelDensity,
+            cladDensity,
+            sodiumDensity,
+            steelDensity,
+            sodiumDensity,
+            steelDensity,
+            sodiumDensity,
+        ]
```

Comment:
The components in order are:
1. Fuel
2. Clad / liner
3. Bond
4. Wire
5. Coolant
6. Duct
7. Intercoolant

It's a little weird that clad and bond are reversed; I think it's because the way the bond is defined in the test reactor for `block_fuel2` is a bit weird/non-physical.

Reply:
I fixed the test reactor block and the ordering is still the same, so I guess this is just how it is. It may just be the order they are defined in the blueprints and not the literal physical ordering of the components.